src/components/ArticleCards.jsx

```diff
@@ -0,0 +1,15 @@
+function ArticleCards({ article }) {
+
+    return (
+      <>
+        <h3>{article.title}</h3>
+        <img className="article-image" src={article.article_img_url} alt={article.title} />
+        <p>Author: {article.author}</p>
+        <p>Topic: {article.topic}</p>
+        <p>Created on: {article.created_at.slice(0, 10)}</p>
```

Comment:
There are some useful libraries out there that would make the created at string nicer to read.